src/main/java/world/inetum/realdolmen/inetumrealJobs/exceptions/CustomGlobalExceptionHandler.java

```diff
@@ -20,15 +21,17 @@
 
     // error handle for @Valid to get personalized messages instead of "Bad Request"
     @Override
-    protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
-                                                                  HttpHeaders headers,
-                                                                  HttpStatus status, WebRequest request) {
+    protected @NonNull
+    ResponseEntity<Object> handleMethodArgumentNotValid(
```

Comment:
I don't like this formatting, the annotation should be on it's own line, before the access modifier